server.js

```diff
@@ -90,6 +88,27 @@ passport.deserializeUser((user, done) => {
   done(null, user);
 });
 
+/**
+ * Authentication check middleware
+*/
+
+const checkAuthentication = (req, res, next) => {
+  console.log('hi checking authentication!');
+  if (req.isAuthenticated()) {
+    console.log('you are good to go sir.')
+    res.send({ isAuthenticated: true });
+    next();
+  } else {
+    console.log('sir. this is VIP.');
+    res.send({ isAuthenticated: false });
+    res.end();
+  }
+};
```

Comment:
eventually we're gonna either want to add to this middleware or create new middleware that checks that that user's UUID in the url string matches up with the session cookie or whatever is being stored from oauth passport